Gemfile

```diff
@@ -54,7 +62,10 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pry', '~> 0.10.4'
```

Comment:
Fair enough. If no one else wants pry as their default console then leave it how it is. (You can always opt out of pry as your console by setting an ENV VAR (DISABLE_PRY_RAILS=1)  if you were to add pry-rails instead)